apps/settings/style/downloads.css

```diff
@@ -93,20 +93,17 @@
 
 /* Text */
 .fileName {
-  padding-top: 1rem;
-  -moz-padding-end: 6rem;
-  padding-bottom: 1rem;
-  -moz-padding-start: 3.5rem;
-  -moz-box-sizing: border-box;
+  padding: 1rem 6rem;
+  box-sizing: border-box;
```

Comment:
thanks for clarify. maybe it's worth to note at https://wiki.mozilla.org/Gaia/CSS_Guidelines#Direction_.28RTL.2FLTR_and_BiDi.29